### DISCUSSION TITLE
--pdf-engine=wkhtmltopdf always fails in pandoc 3.6

### DISCUSSION THREAD

**Discussion by Choonholic** [jgm/pandoc#10469] [Q&A]
**To do:**
Convert markdown document to PDF applied style.css

**Prerequisites**
Pandoc 3.6
wkhtmltopdf 0.12.6

**Reproducing:**
```
pandoc source.md -f markdown -o dest.pdf --pdf-engine=wkhtmltopdf
```
**Symptoms:**
Pandoc 3.5 worked perfectly.

But, Pandoc 3.6 always failed from 1st step like below.

```
Error: Failed loading page file://toPC54C.source (sometimes it will work just to ignore this error with --load-error-handling ignore)
Exit with code 1, due to unknown error.
Error producing PDF.`
```

Unfortunately, `--load-error-handling ignore` did not help.

**Additional Question**
Any alternative solution able to apply styles (font-family, size, page margins and so on.) to markdown when converting?

Thanks.

**Comment by jgm** [comment]
Closing this because you already opened an issue on this.
